Homework_2/task_2.py

```diff
@@ -0,0 +1,19 @@
+# Напишите программу, которая принимает две строки вида “a/b” - дробь с числителем и знаменателем. Программа должна возвращать сумму и произведение* дробей.
+# Для проверки своего кода используйте модуль fractions.
+
+import fractions
+
+f_frac = (str(input("Введите первую дробь в формате a/b ")))
+s_frac = (str(input("Введите вторую дробь в формате a/b ")))
+first_check = fractions.Fraction(f_frac)
+second_check = fractions.Fraction(s_frac)
+f_frac_list = f_frac.split('/')
+s_frac_list = s_frac.split('/')
```

Comment:
Тут можно сразу распаковать в 2 переменные с переводом в `int`, которые будут хранить числитель и знаменатель. Это избавит вас от необходимости ниже возвращать значение по индексам и явно приводить к целочисленному типу каждый раз.
Например так
```suggestion
numerator_1, denominator_1 = map(int, f_frac.split('/'))
numerator_2, denominator_2 = map(int, s_frac.split('/'))
```